test/clj/cider/nrepl/plugin_test.clj

```diff
@@ -0,0 +1,42 @@
+(ns cider.nrepl.plugin-test
+  (:require [cider-nrepl.plugin :refer :all]
+            [cider.nrepl.version :refer [version-string]]
+            [clojure.test :refer :all]))
+
+
+(def expected-output
+  {:dependencies
+   [['org.clojure/clojure]
+    ['cider/cider-nrepl version-string]],
+   :repl-options
+   {:nrepl-middleware
+    '[cider.nrepl/wrap-apropos
+      cider.nrepl/wrap-classpath
+      cider.nrepl/wrap-complete
+      cider.nrepl/wrap-debug
+      cider.nrepl/wrap-enlighten
+      cider.nrepl/wrap-format
+      cider.nrepl/wrap-info
+      cider.nrepl/wrap-inspect
+      cider.nrepl/wrap-macroexpand
+      cider.nrepl/wrap-slurp
+      cider.nrepl/wrap-ns
+      cider.nrepl/wrap-out
+      cider.nrepl/wrap-content-type
+      cider.nrepl/wrap-slurp
+      cider.nrepl/wrap-profile
+      cider.nrepl/wrap-refresh
+      cider.nrepl/wrap-resource
+      cider.nrepl/wrap-spec
+      cider.nrepl/wrap-stacktrace
+      cider.nrepl/wrap-test
+      cider.nrepl/wrap-trace
+      cider.nrepl/wrap-tracker
+      cider.nrepl/wrap-undef
+      cider.nrepl/wrap-version
+      cider.nrepl/wrap-xref]}})
+
+(deftest version-checks
```

Comment:
I guess you can add some test(s) about the normal scenarios as well.

Reply:
Done as well